### PR TITLE
refresh landing

### PR DESCRIPTION
**Pull request by irontitan76**
## Purpose of this pull request

https://github.com/SumoLogic/sumologic-documentation/pull/946

https://user-images.githubusercontent.com/10472920/202571961-708969b2-5500-425b-9728-08bd65b7ac87.mov

Screen Shot 2022-11-17 at 4 21 02 PM
Screen Shot 2022-11-17 at 4 20 00 PM

Issue number: 

N/A

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [x] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by irontitan76** [comment]
Update:

https://user-images.githubusercontent.com/10472920/202599354-de75b028-ebf1-4ba0-bf6e-5947220484ac.mov

File: docusaurus.config.js [hunk 1]
```diff
@@ -13,62 +13,102 @@ const cidRedirects = JSON.parse(fs.readFileSync('cid-redirects.json').toString()
...
   projectName: 'sumologic-documentation', // Usually your repo name.
-  stylesheets: [
-    'https://fonts.googleapis.com/css?family=Material+Icons',
-  ],
-  scripts: [
-    {
-      src: 'https://js.sitesearch360.com/plugin/bundle/3113.js',
-      async: true,
-    },
-  ],
```
Comment (kimsauce) [inline docusaurus.config.js:34]: stet -- this is our search engine plugin code

File: docusaurus.config.js [hunk 2]
```diff
@@ -13,62 +13,102 @@ const cidRedirects = JSON.parse(fs.readFileSync('cid-redirects.json').toString()
...
-  trailingSlash: true,
   baseUrl: process.env.BASE_URL || "/",
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
   favicon: 'https://www.sumologic.com/favicon.ico',
   organizationName: 'sumologic', // Usually your GitHub org/user name.
   projectName: 'sumologic-documentation', // Usually your repo name.
-  stylesheets: [
-    'https://fonts.googleapis.com/css?family=Material+Icons',
-  ],
```
Comment (kimsauce) [inline docusaurus.config.js:28]: Will removing this break the fonts anywhere else?

Comment (irontitan76) [reply docusaurus.config.js:28]: It's not removed! It's down below, alphabetized.

Comment (kimsauce) [reply docusaurus.config.js:28]: oh I see -- thank you!

File: package.json [hunk 3]
```diff
@@ -25,7 +25,6 @@
     "@docusaurus/plugin-sitemap": "2.2.0",
     "@docusaurus/preset-classic": "2.2.0",
     "@docusaurus/theme-classic": "2.2.0",
-    "@docusaurus/theme-search-algolia": "2.2.0",
```
Comment (kimsauce) [inline package.json:28]: I'd like to leave this here -- Algolia is coming shortly

File: docusaurus.config.js (hunk 2 above)
Comment (kimsauce) [reply docusaurus.config.js:28]: Hi @irontitan76 ! I super appreciate your work re-ordering this doc, and totally understand, but I think we will keep the order as-is. Aside from this, were there any other additions/deletions in this file you wanted to make?

**Review by kimsauce — CHANGES_REQUESTED** [review]
Thank you so much, @irontitan76 ! I just had two requests.

File: package.json (hunk 3 above)
Comment (irontitan76) [reply package.json:28]: It's unnecessary regardless, thus the removal! I'd love to have Algolia here—so much better. 

Screen Shot 2022-11-20 at 3 31 09 PM

File: docusaurus.config.js (hunk 2 above)
Comment (irontitan76) [reply docusaurus.config.js:28]: No problem at all, but is there a reasoning behind the order or lack thereof?

I'm a big proponent of code organization and standardization—probably one of the biggest ones around. In my opinion and although a small change, this alphabetization leads to an improved developer experience and simpler perusal of the configuration to find the value to update or verify. The maintainers of Docusaurus obviously don't agree hence the lack of organization here. That said, I'm not a maintainer of this repository, so I'll relent. 

As far as the changes, I don't believe I made any explicit ones here.

File: src/css/infirma.scss [hunk 4]
```diff
@@ -1,343 +1,343 @@
-:root {
```
Comment (irontitan76) [inline src/css/infirma.scss:1]: I didn't mean to commit these commented out styles. I was just looking at whether they were necessary—I don't believe so, but I wasn't going to remove them...yet.

File: package.json (hunk 3 above)
Comment (kimsauce) [reply package.json:28]: Oh awesome, thanks for confirming!

File: docusaurus.config.js (hunk 2 above)
Comment (kimsauce) [reply docusaurus.config.js:28]: I completely agree with your reasoning. But for this specific doc, the ordering aligns with the out-of-the-box, industry standard `docusaurus.config.js` sort order. As I've been working and collaborating in Docusaurus for years (and currently the sole maintainer), I can recite it in my sleep ha. I will take some of your re-ordering suggestions though!

You can check out some examples here:
https://github.com/ipavlic/apex-fp/blob/master/website/docusaurus.config.js
https://github.com/cryptodevhub/site/blob/main/docusaurus.config.js
https://github.com/ariga/atlas/blob/master/doc/website/docusaurus.config.js
https://github.com/webdriverio/webdriverio/blob/main/website/docusaurus.config.js

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hi @irontitan76, many thanks again for your help. Was checking in to see how this is going.

**docusaurus.config.js:** stet (see my comment)

**src/pages/index.tsx:** I'd like to go with what you have [here](https://user-images.githubusercontent.com/10472920/202572499-dcefb9fc-5d19-4e5b-9f12-50de9ab86f01.png), with two tweaks:
- Flip order of assets so that the text/buttons are on the left, graphic is on the right
- Top-align background image so that it looks like this<br/> Screen Shot 2022-11-28 at 8 09 05 PM

**src/theme/Footer/index.tsx**:
- Reduce row spacing (between lines)
- Restore external link indicator icons ([here's how I had it before](https://github.com/SumoLogic/sumologic-documentation/blob/84ef84d9e558a131e48ed09058e024ce51fcc312/src/theme/Footer/index.tsx))
- Match font color to primary dark mode font color

**src/css/sumo.scss:** What do these changes accomplish?

**Comment by irontitan76** [comment]
I'll revert the docusaurus config and update the Footer, but can we get this merged and iterate with any future changes in followup PRs?

**Comment by kimsauce** [comment]
> I'll revert the docusaurus config and update the Footer, but can we get this merged and iterate with any future changes in followup PRs?

yup!

**Comment by irontitan76** [comment]
> src/css/sumo.scss: What do these changes accomplish?

This gets rid of unnecessary styles and adds some spacing for table of contents.

Before:
Screen Shot 2022-11-29 at 4 32 38 PM

After: 
Screen Shot 2022-11-29 at 4 33 06 PM

**Comment by kimsauce** [comment]
> > src/css/sumo.scss: What do these changes accomplish?
> 
> This gets rid of unnecessary styles and adds some spacing for table of contents.

Oh I see - thanks! 

Related item: I also wanted to add is the ability to click on a topic to expand. Currently, you have to click the arrow, but clicking the topic doesn't do anything. Is that something I could do?
menu-bar

**Comment by irontitan76** [comment]
> Related item: I also wanted to add the ability to click on a topic to expand. Currently, you have to click the arrow, but clicking the topic doesn't do anything. Is that something I could do?

The sidebar is somewhat managed by docusaurus but I'm sure it could be updated somehow. I or we would have to look into it in a subsequent PR.

**Comment by irontitan76** [comment]
> Hi @irontitan76, many thanks again for your help. Was checking in to see how this is going.
> 
> **docusaurus.config.js:** stet (see my comment)
> 
> **src/pages/index.tsx:** I'd like to go with what you have [here](https://user-images.githubusercontent.com/10472920/202572499-dcefb9fc-5d19-4e5b-9f12-50de9ab86f01.png), with two tweaks:
> 
> * Flip order of assets so that the text/buttons are on the left, graphic is on the right
> * Top-align background image so that it looks like this Screen Shot 2022-11-28 at 8 09 05 PM
> 
> **src/theme/Footer/index.tsx**:
> 
> * Reduce row spacing (between lines)
> * Restore external link indicator icons ([here's how I had it before](https://github.com/SumoLogic/sumologic-documentation/blob/84ef84d9e558a131e48ed09058e024ce51fcc312/src/theme/Footer/index.tsx))
> * Match font color to primary dark mode font color
> 
> **src/css/sumo.scss:** What do these changes accomplish?

Updated.

File: src/css/sumo.scss [hunk 5]
```diff
@@ -296,6 +296,27 @@ html[data-theme='light'] {
 }
 
 // SIDEBAR Links
+.menu__list-item {
+  .menu__link {
+    padding: 8px 16px;
```
Comment (kimsauce) [inline src/css/sumo.scss:301]: reduce spacing

```suggestion
    padding: 5px 10px;
```

File: src/theme/Footer/index.tsx [hunk 6]
```diff
@@ -91,35 +114,57 @@ export const Footer = () => {
...
+              sx: {
+                cursor: 'pointer',
+                '&:hover': {
+                  color: '#0045BE',
+                },
+              }
             },
             {
               alt: 'Sumo Logic Twitter',
+              color: 'common.white',
```
Comment (kimsauce) [inline src/theme/Footer/index.tsx:152]:
```suggestion
              color: '#e3e3e3',
```

File: src/theme/Footer/index.tsx [hunk 7]
```diff
@@ -91,35 +114,57 @@ export const Footer = () => {
...
               href: 'https://twitter.com/SumoLogic',
-              src: '/img/twitter-logo.png',
-              width: 25,
+              icon: faTwitter,
+              size: 'lg',
+              sx: {
+                cursor: 'pointer',
+                '&:hover': {
+                  color: '#0045BE',
```
Comment (kimsauce) [inline src/theme/Footer/index.tsx:159]: The hover color looks great! Thanks for doing.

File: src/theme/Footer/index.tsx [hunk 8]
```diff
@@ -69,16 +79,29 @@ export const Footer = () => {
...
+                    <ListItemIcon
+                      sx={{
+                        minWidth: 'auto',
+                        ml: 1,
+                        '& svg': {
+                          color: 'common.white',
+                        }
+                      }}
+                    >
+                      <IconExternalLink />
```
Comment (kimsauce) [inline src/theme/Footer/index.tsx:102]: Thanks for adding this back in. Is it possible to get it back to the out-of-box look, as we had it before? 

Currently looks like this -- 
Screen Shot 2022-12-01 at 2 06 07 PM

This is how we'd like it (no gap) --
Screen Shot 2022-12-01 at 2 04 11 PM

**Review by kimsauce — CHANGES_REQUESTED** [review]
just had some small tweaks!

File: src/pages/index.tsx [hunk 9]
```diff
@@ -1,436 +1,303 @@
...
+        }}
+        height={{
+          md: 'auto',
+          xs: '100%',
+        }}
+        py={4}
+        textAlign='center'
+      >
+        <Container maxWidth='lg'>
+          <Grid
```
Comment (kimsauce) [inline src/pages/index.tsx:68]: - Button font style doesn't quite match the original
- On hover, white border should disappear

Should look like this:
Screen Shot 2022-12-01 at 2 25 55 PM

File: src/theme/Footer/index.tsx (hunk 8 above)
Comment (irontitan76) [reply src/theme/Footer/index.tsx:102]: How about show the icon on hover?

File: src/css/sumo.scss (hunk 5 above)
Comment (irontitan76) [reply src/css/sumo.scss:301]: I can definitely update this. That said, the 8px grid is a pretty formidable concept in UI/UX design and development.
https://spec.fm/specifics/8-pt-grid (8pt grid section)

That said, I'll update.

File: src/theme/Footer/index.tsx (hunk 8 above)
Comment (kimsauce) [reply src/theme/Footer/index.tsx:102]: We'd like to keep it static. We had it like Docusaurus has in their footer: https://docusaurus.io.

**Comment by irontitan76** [comment]
> Related item: I also wanted to add the ability to click on a topic to expand. Currently, you have to click the arrow, but clicking the topic doesn't do anything. Is that something I could do? 

In a follow-up PR, sure.

File: src/pages/index.tsx (hunk 9 above)
Comment (irontitan76) [reply src/pages/index.tsx:68]: What specifically about the font style?

https://user-images.githubusercontent.com/10472920/205726032-c8f2592b-eac2-4d07-8ccf-873198f1b19a.mov

File: src/theme/Footer/index.tsx [hunk 10]
```diff
@@ -69,16 +79,29 @@ export const Footer = () => {
...
                   >
                     {label}
                   </ListItemText>
+                  {!isInternalUrl(href) && (
+                    <ListItemIcon
+                      sx={{
+                        minWidth: 'auto',
+                        ml: 1,
+                        '& svg': {
+                          color: 'common.white',
```
Comment (kimsauce) [inline src/theme/Footer/index.tsx:98]:
```suggestion
                          color: '#e3e3e3',
```

**Review by kimsauce — APPROVED** [review]

File: src/pages/index.tsx (hunk 9 above)
Comment (kimsauce) [reply src/pages/index.tsx:68]: [Created follow-up issue for this](https://github.com/SumoLogic/sumologic-documentation/issues/1151)

**Pull request merged**